server/datastore/migrate/migrate.go

```diff
@@ -110,6 +127,13 @@ var repoTokenUpdate = `
 UPDATE repos SET repo_token = '';
 `
 
+var repoBuildNumberColumn = `
+ALTER TABLE repos ADD COLUMN repo_build_number INTEGER
```

Comment:
I would prefer to calculate this value on-the-fly, instead of storing a reference in the repository table. Let's instead use `select max(build_number) from commits where repo_id = ?`. This is what we've done with the drone.io service (different codebase) which has worked really well